tasks/main.yml

```diff
@@ -25,7 +25,10 @@
   args:
     creates: /etc/php5/cli/conf.d/20-mcrypt.ini
 
-- name: Copy php-fpm configuration
+- name: Copy php-fpm php.ini configuration
+  template: src=php.ini dest=/etc/php5/fpm/
```

Comment:
Try running `phpinfo()` in an otherwise blank `.php` file, then hitting it in a web browser, to see what `ini` file should be used.